.github/actions/test-linux/action.yml

```
@@ -18,10 +21,16 @@ runs:
         export PDO_DBLIB_TEST_USER="pdo_test"
         export PDO_DBLIB_TEST_PASS="password"
         export SKIP_IO_CAPTURE_TESTS=1
+        export TEST_PHP_JUNIT=junit.out.xml
         sapi/cli/php run-tests.php -P -q ${{ inputs.runTestsParameters }} \
           -j$(/usr/bin/nproc) \
-          -g FAIL,XFAIL,BORK,WARN,LEAK,XLEAK,SKIP \
+          -g FAIL,BORK,LEAK,XLEAK \
           --offline \
           --show-diff \
           --show-slow 1000 \
           --set-timeout 120
+    - uses: actions/upload-artifact@v3
+      if: always() && inputs.testArtifacts != null
```

Can't that be simplified:
```suggestion
      if: inputs.testArtifacts != null
```

---

@cmb69 Honestly, I don't know. `always()` is used to run jobs even if the predecessors have failed. I'm not sure if there's something magic in it or if it's literally just a `return true;`. The documentation doesn't seem to specify. I can test it though.

---

Ah, okay, then leave it as is. :)